joblib/externals/copy_dill.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# Do a local install of dill.
+
+pip install --target=. dill
+rm -rf *.egg-info
+find dill -name "*.py" | xargs sed -i.bak "s/from dill\(\.\)\{0,1\}/from ./"
```

Comment:
Actually don't listen to me, but put the comment back from copy_joblib.sh

# Note: BSD sed -i needs an argument unders OSX
# so first renaming to .bak and then deleting backup files